.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 dist: xenial
 node_js:
-  - "lts/*"
+  - "12"
```

Comment:
@rcebulko I'm guessing this happened because of the `renovate.json` change you did?

Reply:
Probably, yep